examples/mnist/src/data.rs

```diff
@@ -26,7 +26,8 @@ impl<B: Backend> Batcher<MNISTItem, MNISTBatch<B>> for MNISTBatcher<B> {
             .map(|item| Data::<f32, 2>::from(item.image))
             .map(|data| Tensor::<B, 2>::from_data(data.convert()))
             .map(|tensor| tensor.reshape([1, 28, 28]))
-            .map(|tensor| tensor / 255)
+            // normalize: make between [0,1] and make the mean =  0.1307 and std = 0.3081
```

Comment:
This seems wrong, what we aims for is a mean of zero and a variance of one. However, I'm not sure it will have any meaningfull performance improvement in the case of MNIST, so I would keep the original normalization for simplicity.

Reply:
Sorry my comment is wrong. It should be the mean =  0 and std =1. I took the adjustment values from Pytorch examples: https://github.com/pytorch/examples/blob/main/mnist/main.py#L122 When they transform to tensor, the [0,255] values are adjusted to [0,1].

Thanks for catching.